toolkit/partner-chains-cli/src/create_chain_spec/mod.rs

```diff
@@ -91,12 +93,12 @@ impl<T: PartnerChainRuntime> CreateChainSpecCmd<T> {
 }
 
 #[allow(missing_docs)]
-#[derive(Debug, Default)]
+#[derive(Debug)]
```

Comment:
There is no `Default` for `<Keys>`. Even if `initial_permissioned_candidates_parsed` is by default empty, this won't compile.